README.md

```diff
@@ -23,18 +20,10 @@ your root schema using `allOf` keyword:
 }
 ```
 
-This will include all fields from Dock schema:
+Where:
 
-- `$schema`: denotes what schema was used to build this particular
-  package. It is required to let recipient know what's inside the package.
-- `$originAddress`: original address of the participant that created
-  the data within this data package. This should remain unchanged during
-  the repackaging process so that the final recipient would know who
-  originally created this data (rather than the address of the user who
-  repackaged the data)
-- `$recipientAddress`: address of the designated recipient. It is needed
-  to ensure the recipient that the data was indeed designated for him.
-- `$createdAt`: time value that denotes when original data was created.
-  This should remain unchanged during the repackaging process to let the
```

Comment:
Why did you remove info about persistency of the fields during repackaging?

Reply:
This belongs to some other document. But adding it back for now, until we have that other doc.